### PR TITLE
WIP: Sampler Example

### PR DESCRIPTION
**Pull request by WPFerg**
Uses a large data set with the low barrel example, with automatic zooms and choice between sampling method.

### CODE REVIEW

File: site/src/examples/sampler/index.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.candlestick.down {
+  fill: #c60;
+}
+
+.bar {
+  stroke: #06c;
+  fill: #9cf;
+}
```
Comment (chrisprice) [inline site/src/examples/sampler/index.css]: Please can you add a newline at the end of the file?

File: site/src/examples/sampler/index.js [hunk 2]
```diff
@@ -0,0 +1,399 @@
+/* global d3:false, fc:false */
+    'use strict';
```
Comment (chrisprice) [inline site/src/examples/sampler/index.js]: The indentation seems to be a bit awry, please could you reformat to use 4 spaces?

**Comment by chrisprice** [comment]
Thanks @WPFerg, I think the example is great and a really compelling visualisation of what sampling is about and the benefits of using it. However, as there's so much commonality between this and the low barrel example (through no fault of your own!) I would like to hold off merging this. 

There's some on-going work to extract the common components of the low-barrel example into the library itself. Once this has been done the amount of code in your example should drop dramatically and it'll be easier to comment on the specifics of your code.

As such I've marked this as a WIP and I've added a couple of high-level comments, but I'll save the rest for later.

**Comment by WPFerg** [comment]
No problem, I've made the indentation changes. I've got another PR coming for one more sampler that I'd like to integrate in to this, so I'll revisit it later.

**Comment by ColinEberhardt** [comment]
@WPFerg I'm going to close this for now - good to see the code being made good use of in your blog post. Once we have a navigator component I'd like to revisit this example and potentially bring it into the project.

**Pull request closed without merging**